lldb/source/Plugins/MemoryHistory/asan/MemoryHistoryASan.cpp

```diff
@@ -174,6 +174,21 @@ HistoryThreads MemoryHistoryASan::GetHistoryThreads(lldb::addr_t address) {
   options.SetAutoApplyFixIts(false);
   options.SetLanguage(eLanguageTypeObjC_plus_plus);
 
+  const auto &target = process_sp->GetTarget();
+  std::unordered_set<ModuleSP> preferred_modules;
+  target.GetImages().ForEach([&](const lldb::ModuleSP &m) {
+    llvm::Regex pattern("libclang_rt.asan_.*_dynamic.dylib");
```

Comment:
@wrotki is this going to hold in the long-term? Will the locally built compiler-rt always be `libclang_rt`?